docroot/sites/now.uiowa.edu/modules/now_core/now_core.module

```diff
@@ -258,3 +280,59 @@ function now_core_article_type_tag_map(string $article_type = '') {
   }
   return $map[$article_type] ?? FALSE;
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function now_core_preprocess_views_view_unformatted(&$variables) {
+
+  $view = $variables['view'];
+  $variables['row_item_classes'] = [];
+  if ($view->current_display === 'page_experts_list') {
+    $style = $view->style_plugin;
+    $rows = $variables['rows'];
+
+    $default_classes = [
+      'card--list',
+      'card--media-left',
+      'card--person-teaser',
+    ];
+
+    $variables['row_item_classes'] = array_merge($default_classes, array_values($variables['row_item_classes']));
+    foreach ($rows as $id => $row) {
+      // Set remaining row_item_classes as attributes on each row.
+      if (isset($variables['row_item_classes'])) {
+        $variables['rows'][$id]['attributes']->addClass($variables['row_item_classes']);
+      }
+      // Get node path to link card with.
+      $nid = $view->result[$id]->_entity->id();
+      $link = \Drupal::service('path_alias.manager')->getAliasByPath('/node/' . $nid);
+
+      // Initialize card_options variable.
+      $variables['rows'][$id]['card_options'] = [
+        'attributes' => $variables['rows'][$id]['attributes'],
+        'media_styles' => ['media--circle', 'media--border'],
+        'media_sizes' => ['card__media--small'],
+        'card_image' => $style->getField($id, 'field_image'),
+        'card_title' => $style->getField($id, 'title'),
+        'card_link_url' => $link,
+        'card_subtitle' => $style->getField($id, 'field_person_position'),
+        'person_email' => $style->getField($id, 'field_person_email'),
+        'person_phone' => $style->getField($id, 'field_person_phone'),
```

Comment:
This is related to my comment about the view template. If we don't care about having the extra markup associated with using the existing `{% block card_meta %}`, the I believe that both of these element should already be render arrays that will print out correctly if set as the value of `card_meta`. I'm not exactly sure, but I think you could do something like this:
```suggestion
        'card_meta' => [
          'email' => $style->getField($id, 'field_person_email'),
          'phone' => $style->getField($id, 'field_person_phone'),
        ],
```